CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ if (WIN32)
 endif()
 option(BUILD_SHARED_LIBS "Build shared libraries (.dll/.so) instead of static ones (.lib/.a)" OFF)
 option(BUILD_TESTING "Build test programs" ON)
+option(maxminddb_BUILD_BINARIES "Build binaries" ON)
+option(maxminddb_INSTALL "Generate the install target" ON)
```

Comment:
Maybe we could have the option names be all uppercase, e.g., `MAXMINDDB_INSTALL`. This would better match what we are doing with `MAXMINDDB_SOVERSION` already and it seems to be the preferred naming scheme for the handful of projects that I looked at.